applications/GeoMechanicsApplication/custom_utilities/input_utilities.cpp

```diff
@@ -0,0 +1,38 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Anne van de Graaf
+//
+
+#include "input_utilities.h"
+#include "containers/model.h"
+#include "utilities/read_materials_utility.h"
+
+
+namespace Kratos
+{
+
+Parameters InputUtilities::ProjectParametersFrom(const std::string& rProjectFilePath)
+{
+    std::ifstream t(rProjectFilePath);
+    std::stringstream buffer;
+    buffer << t.rdbuf();
+    Parameters projFile{buffer.str()};
+    return projFile;
+}
+
+void InputUtilities::AddMaterialsFrom(const std::string& rMaterialFilePath,
```

Comment:
If ProjectParametersFrom then why AddMaterialsFrom. Do you see the material parameter .json file as additional to the ProjectParameters.json

Reply:
The `ProjectParametersFrom` function just produces a `Parameters` object from the given JSON file. On the other hand, `AddMaterialsFrom` will add material data to the given model and not produce a new object. I do not intend to change the function names, unless you can provide me with better alternatives. Thanks.